client/app/controllers/subscribe.js

```diff
@@ -0,0 +1,92 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import fetch from 'fetch';
+import ENV from 'labs-zap-search/config/environment';
+
+const tlds = ['.com', '.gov', '.edu', '.net', '.org'];
```

Comment:
Instead of keeping track of our own list of tlds and using that as validation, can we use the same regex for emails already present in `subscription-form.js`? I'd rather we just check that the email address matches a _pattern_ than check against a list of tlds. If what they enter after `.` isn't an actual tld, they should get the same behavior as any other invalid email address.

Reply:
See conversation below.